docs/tableconfig_schema.rst

```diff
@@ -113,6 +113,25 @@ The ``segmentsConfig`` section has information about configuring the following:
       "segmentAssignmentStrategy": "BalanceNumSegmentAssignmentStrategy"
     },
 
+* Completion Config
+
+  You can also add a ``completionConfig`` section under the ``segmentsConfig`` section. Completion config holds information related to realtime segment completion. There's just 1 field in this config as of now, which is the ``completionMode``. The value of the ``completioMode`` decides how non-winner servers should replace the completed segment during realtime segment completion. Currently, the supported values are
```

Comment:
s/1/one/

Please add a couple of sentences on winner or non-winner (better use commit leader and commit follower) in the architecture section and then add a sentence here to refer to the arch section for what these mean. You can also point to the realtime design document  segment completion protocol section. for more read.

Reply:
Added to the architecture section.